setup.py

```diff
@@ -1,6 +1,9 @@
 from setuptools import find_packages, setup
 
-install_requires = ["wagtail>=2.3,<2.10", "django-flags>=4.2,<5.0"]
+install_requires = [
+    "wagtail~=2.0",
```

Comment:
I'd like to continue pinning versions of Wagtail to ensure we are compatible when new ones release. There's no guarantee that wagtail-flags will be compatible with 2.0 *and* 2.12.

Reply:
Removed wagtail change to install_requires as this is out of scope for this PR